include/NAS2D/Resources/Music.h

```diff
@@ -20,16 +20,16 @@ namespace NAS2D {
  *  \class Music
  *  \brief Music Resource.
  */
-class Music: public Resource
+class Music : public Resource
 {
 public:
-	Music();
+	Music() = default;
 	Music(const std::string& filePath);
 
 	Music(const Music& rhs);
 	Music& operator=(const Music& rhs);
 
-	~Music();
+	virtual ~Music();
```

Comment:
There are no other virtual member functions. Maybe we don't want to make the destructor virtual.

Do we even want to explicitly declare a destructor here?

Reply:
Music is derived from Resource which itself has declared its destructor `virtual`. Since the virtual-ness of a function is automatically inherited by derived types it's not *required* that Music's destructor be declared virtual but it makes it more obvious that it is virtual. The `override` keyword not withstanding.

Reply:
Ahh, that makes more sense.

I suppose if we declare it, we should use `override` then. Though if it's empty, we're not really overriding anything.

Reply:
Opened issue #400 to track changes to use `override` keyword.

On closer inspection, the destructor is being used here, so does need to be declared. I may have had this confused with another class which had an explicit empty destructor definition.